src/CommunityToolkit.Maui.Core/Handlers/Popup/PopupHandler.net.cs

```diff
@@ -13,6 +13,7 @@ public partial class PopupHandler : Microsoft.Maui.Handlers.ElementHandler<IPopu
 	/// <param name="result">The result that should return from this Popup.</param>
 	public static void MapOnClosed(PopupHandler handler, IPopup view, object? result)
 	{
+		view.HandlerCompleteTCS.TrySetResult();
```

Comment:
Shouldn't this be empty or throwing an `NotImplementedException`? Since this one is if anyone tries to use it on an unsupported platform.

Reply:
Done! 

For consistency, I updated the other methods in `PopupHandler.net.cs` to throw a `NotSupportedException` too